bot.py

```diff
@@ -159,12 +160,22 @@ def redis(self):
     def data(self):
         return self.get_cog("Data").instance
 
+    @property
+    def lang(self):
+        return self.get_cog("Lang").fluent
+
     @property
     def sprites(self):
         return self.get_cog("Sprites")
 
     # Other stuff
 
+    def _(self, message_id: str, **kwargs: dict[str, typing.Any] | None) -> str:
```

Comment:
I don't think this is the correct typing for `**kwargs`

Reply:
It should be according to `fluent.runtime` source, unless I'm missing something?

```py
def format_value(self, msg_id: str, args: Union[Dict[str, Any], None] = None) -> str:
```

Reply:
There's wrappers for Fluent types in `fluent.runtime.types`, like `FluentType` and `FluentNumber`, but those seem to be data wrappers that are used internally (?) by the library.